mmv1/third_party/terraform/tests/resource_compute_instance_test.go.erb

```diff
@@ -4473,6 +4535,55 @@ resource "google_compute_instance" "foobar" {
 `, disk, family, family, instance)
 }
 
+func testAccComputeInstance_networkPerformanceConfig(disk string, image string, instance string) string {
+	return fmt.Sprintf(`
+data "google_compute_image" "my_image" {
+  family  = "debian-9"
+  project = "debian-cloud"
+}
+
+resource "google_compute_disk" "foobar" {
+  name  = "%s"
+  zone  = "us-central1-a"
+  image = data.google_compute_image.my_image.self_link
+}
+
+resource "google_compute_image" "foobar" {
+  name              = "%s"
+  source_disk       = google_compute_disk.foobar.self_link
+  guest_os_features {
+    type = "GVNIC"
+  }
+  guest_os_features {
+    type = "VIRTIO_SCSI_MULTIQUEUE"
+  }
+}
+
+resource "google_compute_instance" "foobar" {
+  name         = "%s"
+  machine_type = "n2-standard-2"
+  zone         = "us-central1-a"
+
+  boot_disk {
+    initialize_params {
+      image = google_compute_image.foobar.self_link
+    }
+  }
+
+  network_interface {
+    network = "default"
+    access_config {
+      // Ephemeral IP
+    }
+  }
+
+  network_performance_config {
+    total_egress_bandwidth_tier = "DEFAULT"
```

Comment:
This is what we settled on to test functionality here since `TIER_1` requires n2-x-32 or larger instances.

Reply:
Thanks for getting this to work!